libtiledbsoma/src/soma/managed_query.h

```diff
@@ -1086,6 +1104,15 @@ bool ManagedQuery::_extend_and_evolve_schema_and_write<std::string>(
     Enumeration enmr,
     ArraySchemaEvolution& se);
 
+template <>
+bool ManagedQuery::_extend_and_evolve_schema<std::string>(
+    ArrowSchema* value_schema,
+    ArrowArray* value_array,
+    std::string column_name,
```

Comment:
Do we mutate the `column_name` or is it safe to pass it by a const reference? (Similarly for the `column_name` parameter in the other new methods.)

Reply:
Sure. General advice I've seen over the years is const references aren't a worthwhile gain for small strings (which these are). However, it doesn't hurt; I'll make them const references.